pkgs/development/python-modules/notus-scanner/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, paho-mqtt
+, poetry
+, psutil
+, pytestCheckHook
+, pythonOlder
```

Comment:
nit: it would be nice to alphabetize

Reply:
Hmm, there is `lib` and the rest is in alphabetical order.